packages/core/src/new-api/internal/module-builder.ts

```diff
@@ -396,8 +436,6 @@ export class IgnitionModuleBuilderImplementation<
       future.dependencies.add(address);
     }
 
-    // TODO: Validate the the runtime value's default type is string
```

Comment:
I want to note that currently, enforcing this the way I did has introduced the problem that module params without default values are considered invalid due to them getting assigned an `any` type by default.

Reply:
Not sure if I understand this. But this TODO, which I thing I wrote, is incorrect, right? Default values can have types other than strings.